pkgs/by-name/dn/dns-over-https/package.nix

```diff
@@ -0,0 +1,40 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildGoModule,
+  nix-update-script,
+}:
+buildGoModule rec {
+  pname = "dns-over-https";
+  version = "2.3.7";
+
+  src = fetchFromGitHub {
+    owner = "m13253";
+    repo = pname;
+    tag = "v${version}";
```

Comment:
Are you sure that the tag has a `v` prefix ?

Reply:
Yes, I am sure.

Reply:
Fair enough.